unityfiles/Assets/Scripts/Game/Enemy/EBTestMoveDown.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Enemy Behavior test move down
+/// test behavior that moves the enemy downwards
+/// </summary>
+
+[RequireComponent (typeof (Enemy))]
+public class EBTestMoveDown : MonoBehaviour, IEnemyBehavior {
+
+    private Enemy enemy;
+    private float velocity;
+
+    /// <summary>
+    /// Start is called on the frame when a script is enabled just before
+    /// any of the Update methods is called the first time.
+    /// </summary>
+    void Start () {
+        this.enemy = this.gameObject.GetComponent<Enemy> ();
+        this.velocity = enemy.EnemyType.speed;
```

Comment:
Skal disse test scriptene være med?

Reply:
tenkte at det kan vere veldig praktisk for folk å ha en mal når de skal lage egene behaviours

Reply:
Lurt. Men kall den `EnemyBehaviorTemplate` eller noe lignende

Reply:
Det er ikke en template det er en test enhet brukt under utvikling enten blir den eller så fjerner jeg den helt. Hvilken av de to er samme for meg